Lecture-4/Task4/Task4/Application.java

```diff
@@ -0,0 +1,78 @@
+package Task4;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Application {
+
+    private static ArrayList<Runner> listNotSorted;
+    private static ArrayList<Runner> listSorted;
+
+    private static void showListOfRunners(ArrayList<Runner> list){
+        for (Runner runner:list) {
+            runner.showRunner();
+        }
+    }
+
+    private static void makeMarathon(){
+        int bestTime;
+        Runner bestRunner = listNotSorted.get(0);
```

Comment:
A better approach is to use either `Comparable` of `Comparator` interfaces and sort list using `Collections.sort(..)` method

Reply:
done